client/packages/common/src/intl/locales/en/common.json

```diff
@@ -55,7 +56,7 @@
   "description.our-soh": "Our Stock on Hand",
   "description.pack-quantity": "Total quantity in number of packs",
   "description.program": "Programs are used to manage groups of items, and replenishment schedules.",
-  "description.remaining-to-supply": "The quantity remaining to be supplied, which is the Supply quantity less the quantity of this item on any linked Outbound Shipment",
+  "description.remaining-to-supply": "The remaining quantity to supply to the customer which will be re-calculated after a shipment has been created",
```

Comment:
I thought the explanation of the calculation was useful - is there a reason to remove?
```
The quantity remaining to be supplied to the customer which will be re-calculated after a shipment has been created. The calculation is 'Supply - Issued'
```

Reply:
No reason, have added it back (: